docs/source/CONTRIBUTE.md

```diff
@@ -1,9 +1,35 @@
-# Contributing to MALA
-
-MALA is open source software and as such always welcomes additions and
-improvements. However, we kindly ask any contributor to adhere to the following
-suggestions in order to keep the overall quality and maintainability of the
-code high.
+# Contributions
+
+MALA is open source software and based on the work of many people. 
```

Comment:
MALA is an open-source software and is built upon the collaborative efforts of many contributors. The MALA team warmly welcomes additional contributions and kindly requests potential contributors to follow the suggested guidelines below to ensure the code's overall quality and maintainability.